bucket/python.json

```diff
@@ -44,23 +50,23 @@
             "idle3"
         ]
     ],
-    "env_add_path": "scripts",
+    "env_add_path": "Scripts",
     "checkver": {
         "url": "https://www.python.org/downloads/windows/",
         "re": "Latest Python 3 Release - Python ([\\d\\.]+)"
     },
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://www.python.org/ftp/python/$version/python-$version-amd64.exe#/py3.exe"
+                "url": "https://www.python.org/ftp/python/$version/python-$version-amd64.exe"
             },
             "32bit": {
-                "url": "https://www.python.org/ftp/python/$version/python-$version.exe#/py3.exe"
+                "url": "https://www.python.org/ftp/python/$version/python-$version.exe"
             }
         },
         "hash": {
             "url": "https://www.python.org/downloads/release/python-$cleanVersion/",
-            "find": "<td><a href=\".*$basename\".*</a></td>\\s+<td>Windows</td>\\s+<td>.*</td>\\s+<td>([A-Fa-f\\d]{32})"
+            "find": "$basename[\\S\\s]+?$md5"
```

Comment:
Will this get correct hash?
If i remember right I made it this strict to match only the correct one hash.

Reply:
Yes, the regex find the next md5 hash after basename.

![checkver](https://user-images.githubusercontent.com/5832170/56673203-6dadd500-66ea-11e9-9fac-7b9e2628c9e2.png)